railgun/manager/manager.go

```diff
@@ -59,12 +59,13 @@ type Config struct {
 	EnableLeaderElection bool
 	LeaderElectionID     string
 	ProbeAddr            string
-	KongURL              string
+	KongAdminURL         string
 	FilterTag            string
 	Concurrency          int
 	KubeconfigPath       string
 	IngressClassName     string
 	AnonymousReports     bool
+	KongWorkspace        string
```

Comment:
Please ensure that we test workspaces before we release beta.

That is: either add an acceptance criterion to #1279 that we test using workspaces, or create an issue on its own and add it to the milestone.

Reply:
Added along with the other Enterprise-specific flag (RBAC token).